docs/console_scripts.rst

```diff
@@ -0,0 +1,40 @@
+Console_scripts
```

Comment:
oh nice... @kcranston was this just a way to document the old stuff before adding the new stuff? somewhere i swear i had some instructions but still haven't been able to find them. will dig some more (if that is the goal!).

Reply:
right - I had notes on how distribute worked as I planned out the new script, so I figured I'd just add them here

Reply:
awesome!! i swear somewhere there are more instructions that tim gave me... one day i'll find them.